Singel node with partially deleted cluster configs won't start up

**Description:**
Single SP node with below as cluster config
```
cluster.config:
  enabled: false

```
will fail to start up with NPE.
```
Caused by: java.lang.NullPointerException
 at org.wso2.carbon.cluster.coordinator.rdbms.internal.RDBMSCoordinationServiceComponent.start(RDBMSCoordinationServiceComponent.java:59)
 ... 28 more
```
Ideally product should be able to start just with setting the configs to false.

**Affected Product Version:**
4.3.0

**Steps to reproduce:**
Delete subsequent part of cluster.config and start the pack

--- a/components/cluster-coordinator/org.wso2.carbon.cluster.coordinator/org.wso2.carbon.cluster.coordinator.rdbms/src/main/java/org/wso2/carbon/cluster/coordinator/rdbms/internal/RDBMSCoordinationServiceComponent.java
+++ b/components/cluster-coordinator/org.wso2.carbon.cluster.coordinator/org.wso2.carbon.cluster.coordinator.rdbms/src/main/java/org/wso2/carbon/cluster/coordinator/rdbms/internal/RDBMSCoordinationServiceComponent.java
@@ -55,7 +55,7 @@ public class RDBMSCoordinationServiceComponent {
         try {
             clusterConfiguration = RDBMSCoordinationServiceHolder.getConfigProvider().
                     getConfigurationObject(ClusterCoordinatorConfigurations.class);
-            if (clusterConfiguration != null) {
+            if (clusterConfiguration != null && clusterConfiguration.isEnabled()) {
                 if (clusterConfiguration.getStrategyConfig().getHeartbeatInterval() < 5000) {
                     log.warn("It is recommended to have the heartbeatInterval value as " + 5000 +
                             " milliseconds or higher than the provided " +

--- a/pom.xml
+++ b/pom.xml
@@ -322,8 +322,8 @@
         <carbon.datasources.version>1.1.8</carbon.datasources.version>
         <commons.logging.version>1.2</commons.logging.version>
         <commons.logging.imp.package.version>[1.2,2.0)</commons.logging.imp.package.version>
-        <guava.version>20.0</guava.version>
-        <guava.bundle.version>20.0.0</guava.bundle.version>
+        <guava.version>31.1-jre</guava.version>
+        <guava.bundle.version>31.1.0.jre</guava.bundle.version>
         <guava.imp.package.version>[20.0.0,21.0.0)</guava.imp.package.version>
         <apache.curator.imp.package.version>[2.6.0,2.9.0)</apache.curator.imp.package.version>
         <com.h2database.version>1.4.187</com.h2database.version>

